prototypes/earth.lua

```diff
@@ -1,4 +1,4 @@
-﻿if not feature_flags.space_travel or not settings.startup['only-apply-to-earth'].value then
+﻿if not mods['space-age'] or not settings.startup['only-apply-to-earth'].value then
```

Comment:
Was this broken before? Or is this just a better way of doing things? If so, do you have some official docs for this?